rosbag2_transport/include/rosbag2_transport/play_options.hpp

```diff
@@ -26,6 +26,7 @@ struct PlayOptions
 public:
   size_t read_ahead_queue_size;
   std::string node_prefix = "";
+  std::string clock_type = "past";
```

Comment:
What values are accepted here? I am thinking this should probably be an enum.

Reply:
Could this be part of the actual clock types being available in ros2? http://docs.ros2.org/eloquent/api/rcl/time_8h.html#a5c734f508ce06aec7974af10ad09d071

Reply:
this was @scoopySHI 's way of getting the simulated clock argument into the core player class. I do agree that this should be done in a better way.

Reply:
Maybe the name clock_type can lead to misunderstand. I'd like to replace it with "time in Header": it suggests which time is in the time stamper in Header (current system time or the time while recording). It has actually nothing to do with clock types in ROS2.